package/yast2-theme.spec

```diff
@@ -127,10 +128,12 @@ rm -rf $RPM_BUILD_ROOT/%{yast_themedir}/openSUSE*
 rm -rf "$RPM_BUILD_ROOT/%{yast_docdir}"
 rm -rf "$RPM_BUILD_ROOT/%{_docdir}/yast2-theme"
 
-mv $RPM_BUILD_ROOT/usr/share/icons/hicolor $RPM_BUILD_ROOT/usr/share/YaST2/theme/SLE/icons/
+mv $RPM_BUILD_ROOT%{yast_themedir}/SLE $RPM_BUILD_ROOT%{yast_themedir}/current
+
+mv $RPM_BUILD_ROOT/usr/share/icons/hicolor $RPM_BUILD_ROOT/usr/share/YaST2/theme/current/icons/
 
 # remove all icons that were not part of RC2 to avoid information leak
```

Comment:
I am not sure if this comment still applies.

Reply:
well, it looks like it remove all icons that is not used. But I am not sure if it applies. I do not know to be honest.

Reply:
Then keep it as it is :)